tests/cypress/forms_spec.js

```diff
@@ -98,11 +104,27 @@ context("Marketo dynamic form", () => {
 
     getIframeBody().find(".rc-anchor-content").click();
 
-    cy.wait(10000); // eslint-disable-line
+    cy.wait(3000); // eslint-disable-line
     cy.get("#mktoForm_1251")
       .submit()
       .should((page) => {
-        expect(page[0].action).to.equal("/marketo/submit");
+        expect(page[0].action).to.include("/marketo/submit");
       });
+    cy.get(".p-modal__close").click();
+  });
+
+  it("should successfully complete download server", () => {
+    cy.visit("/download/server/s390x");
+
+    cy.get('input[name="firstName"]').type("Test");
```

Comment:
I'd suggest using `findByLabelText` (so looking for an actual label that the user will be able to read) instead of a input's name. You want to avoid testing implementation details as much as possible.

```suggestion
    cy.findByLabelText(/First name:/).type("Test");
```